storage/constraints.go

```diff
@@ -62,7 +62,7 @@ func ParseConstraints(s string) (Constraints, error) {
 		}
 		if IsValidPoolName(field) {
 			if cons.Pool != "" {
-				logger.Debugf("pool name is already set to %q, ignoring %q", cons.Pool, field)
+				return cons, errors.NotValidf("pool name is already set to %q, cannot set as %q", cons.Pool, field)
```

Comment:
`return cons, errors.NotValidf("pool name is already set to %q, new value %q", cons.Pool, field)` 
will result in messages like:
`pool name is already set to "pool1", new value "pool2" not valid`
which reads a bit better.

Reply:
Cheers, Done & Done